### PR TITLE
New pin and unpin icons for Widget Panels (bp #4158)

### PR DESCRIPTION
**Pull request by NancyMcCallB**
Backport to provide the new pin/unpin icons in 3.3 for DR testing. (Manual backport)

### CODE REVIEW

File: common/changes/@bentley/imodelbank-client/ui-backport-pin-unpin-icons_2022-09-07-19-51.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```
Comment (aruniverse) [inline common/changes/@bentley/imodelbank-client/ui-backport-pin-unpin-icons_2022-09-07-19-51.json:2]: you should delete this file

**Review by pmconne — APPROVED** [review]

**Review by aruniverse — APPROVED** [review]
wrong change file

`rush change -b origin/release/3.3.x`

File: common/changes/@bentley/imodelbank-client/ui-backport-pin-unpin-icons_2022-09-07-19-51.json (hunk 1 above)
Comment (pmconne) [reply common/changes/@bentley/imodelbank-client/ui-backport-pin-unpin-icons_2022-09-07-19-51.json]: bye.

File: ui/appui-layout-react/src/appui-layout-react/widget/PinToggle.tsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ export const PinToggle = React.memo(function PinToggle() { // eslint-disable-lin
       }}
       title={panelState.pinned ? unpinPanelTitle : pinPanelTitle}
     >
-      <Icon iconSpec={panelState.pinned ? <SvgUnPin /> : <SvgPin />} />
+      <Icon iconSpec={panelState.pinned ? <SvgPin /> : <SvgUnPin />} />
```
Comment (GerardasB) [inline ui/appui-layout-react/src/appui-layout-react/widget/PinToggle.tsx:57]: LGTM for a backport. Should we consider rewording this back to:
`pinned ? <Unpin /> : <Pin />` OR `pinned ? <Pinned /> : <Unpinned />`

**Review by GerardasB — APPROVED** [review]

File: ui/appui-layout-react/src/appui-layout-react/widget/PinToggle.tsx (hunk 2 above)
Comment (NancyMcCallB) [reply ui/appui-layout-react/src/appui-layout-react/widget/PinToggle.tsx:57]: We can, but it's probably better if these icons go into itwinui-icons and we use them from there.

**Pull request merged**